SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTestRequireHandler.cpp

```diff
@@ -102,9 +108,21 @@ FString GenerateStatusMessage(bool bPassed, FString Received, FString Expected,
 	}
 }
 
-SpatialFunctionalTestRequireHandler::SpatialFunctionalTestRequireHandler()
-	: NextOrder(0)
+bool SpatialFunctionalTestRequireHandler::RequireValid(const UObject* Object, const FString& Msg)
 {
+	bool bPassed = IsValid(Object);
+	FString StatusMsg;
+	if (bPassed)
+	{
+		StatusMsg = FString::Printf(TEXT("Received valid actor %s as expected."), *Object->GetName());
+	}
+	else
+	{
+		StatusMsg = FString::Printf(TEXT("Unexpectedly received invalid Actor %s."), *GetNameSafe(Object));
```

Comment:
actor is capitalized here, but with lower-case "A" above, bothers me more than it should

Reply:
changed